### ISSUE TITLE
On Chrome 60 RecordRTC.stopRecording not working

### ISSUE THREAD

**Issue by rubenkuipers** [muaz-khan/RecordRTC#311]
Seems like Chrome 60.0.3112.101 causes a bug when you try to call the function ``RecordRTC.stopRecording``. Response: ``GET blob:https://www.webrtc-experiment.com/ce6bc182-7a38-414d-dsk78-c87d30f7f02b 416 (Requested Range Not Satisfiable)``

Also appears in the demo on https://www.webrtc-experiment.com/RecordRTC/
Any idea why this happens? Just upgraded to the new version of RecordRTC also.

**Comment by muaz-khan** [comment]
Can you record only video tracks (ignore audio) or pass `timeSlice:1000` parameter?

Both tracks must be active. Are you able to listen your voice using `<video>` element?

You can try vp9: `mimeType: 'video/webm\;codecs=opus,vp9'`

**Comment by michalstepien** [comment]
Yes, look  this muaz-khan: https://www.webrtc-experiment.com/RecordRTC/Canvas-Recording/record-canvas-drawings.html

**Comment by zslucky** [comment]
Same issue, but looks like I'm correct before. but these days suddenly broken... no code changes, but got the same error `416 (Requested Range Not Satisfiable)`.

**Comment by Shashwat986** [comment]
I found this issue by searching for the same problem. I'm not using this package, but I feel this is a chrome settings issue.

I fixed it by going to `chrome://flags` and enabling `Experimental Web Platform features`. Does that work for you guys?

**Comment by kuiperr005** [comment]
It looks like this issue is fixed in the latest version of Chrome 62.0, although in my case :)

**Comment by zslucky** [comment]
Also fixed by upgrade the chrome from 61 -> 62.
https://stackoverflow.com/questions/46922911/getusermedia-use-audio-only-will-be-closed-by-itself-in-few-seconds-on-mac-chrom/46926314#46926314
